vslib/inc/MACsecForwarder.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "swss/sal.h"
+#include "swss/selectableevent.h"
+
+#include <string>
+#include <memory>
+#include <thread>
+
+namespace saivs
+{
+    class MACsecForwarder
+    {
+    public:
+        MACsecForwarder(
+            _In_ const std::string &macsecInterfaceName,
+            _In_ int tapfd);
+
+        virtual ~MACsecForwarder();
+
+        int get_macsecfd() const;
+
+        void forward();
+
+    private:
+        int m_tapfd;
+        int m_macsecfd;
+
+        const std::string m_macsec_interface_name;
```

Comment:
try keep naming members consistent like m_macsecInterfaceName, m_runThread, m_exitEvent

please revisit other classes  to see convention